cns/restserver/util.go

```diff
@@ -600,6 +600,16 @@ func (service *HTTPRestService) getNetworkContainerDetails(networkContainerID st
 	return containerDetails, containerExists
 }
 
+// areNCsPresent returns true if NCs are present in CNS, false if no NCs are present
+func (service *HTTPRestService) areNCsPresent() bool {
+	if len(service.state.ContainerStatus) == 0 {
+		if len(service.state.ContainerIDByOrchestratorContext) == 0 {
```

Comment:
can you one-line this with an `||`? or maybe it should be a switch?
```suggestion
	if len(service.state.ContainerStatus) == 0 || len(service.state.ContainerIDByOrchestratorContext) == 0 {
```

Reply:
I would probably `&&` it instead, so that both things need to be empty for it to say it's empty

Reply:
I put an `&&` @rbtr so it's one line now